src/components/Breadcrumbs/Breadcrumbs.stories.mdx

```diff
@@ -113,6 +113,25 @@ denoted by an `aria-current` attribute value of "page".
   </DSProvider>
 </Canvas>
 
+<Canvas>
+  <DSProvider>
+    <Breadcrumbs
+      breadcrumbsData={[
+        { url: "#", text: "Parent with a Long Name" },
+        { url: "#", text: "Child with an Even Longer Name" },
+        {
+          url: "#",
+          text: "Grandchild with an Exceptionally Long Name. Long titles shorten to 60 characters, an ellipsis will be added, and that the Tooltip is used to see the entire text characters "
+        },
+        {
+          url: "#",
+          text: "Great-Grandchild with the Longest Name",
```

Comment:
The "Grandchild" text is fine but also make this great-grandchild long text as well. Then the example will show what happens when _two_ titles are extremely long.